config/default.go

```diff
@@ -234,8 +234,6 @@ Mode = "PessimisticProof"
 CheckStatusCertificateInterval = "5m"
 RetryCertAfterInError = false
 GlobalExitRootL2="{{L2Config.GlobalExitRootAddr}}"
-# Don't send certificate over 80% of the epoch
-MaxEpochPercentageAllowedToSendCertificate=80
```

Comment:
Remove this param from the [aggsender.md](https://github.com/agglayer/aggkit/blob/8e28051fa30de5952a3aa6a88b7818d6e9c8ccf4/docs/aggsender.md#L171) as well.